orchestrator/tests/nl2sql_eval/harness.py

```
@@ -0,0 +1,147 @@
+    """Run every seeded question through ``generate_fn`` and score by execution
+    accuracy against the golden SQL."""
+    conn = build_seeded_db()
+    schema = introspect_schema(conn)
+    report = EvalReport()
+
+    for q in load_questions():
+        report.total += 1
+        qr = QuestionResult(id=q["id"], question=q["question"], golden_sql=q["golden_sql"])
+        try:
+            golden_rows = run_sql(conn, q["golden_sql"])
+        except Exception as e:  # a broken golden is a harness bug, surface it
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)